lib/foreman_openscap/engine.rb

```diff
@@ -155,7 +155,7 @@ class Engine < ::Rails::Engine
         parameter_filter Hostgroup, :openscap_proxy_id, :openscap_proxy
         parameter_filter Log, :result
 
-        if ForemanOpenscap.with_remote_execution?
+        if ForemanOpenscap.with_remote_execution? && Gem::Version.new(ForemanRemoteExecution::VERSION) >= Gem::Version.new('1.2.3')
```

Comment:
Is there any reason to have the feature if the version is less than 1.2.3? Because 1.2.2 and earlier can register features just not host buttons.